src/lib/CreateRoom.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  function createRoom(): void {
+    // call create-room API
```

Comment:
If this function is going to call the API, I'd go ahead and mark it `async` even when stubbing.